src/core/common/Sidebar/getMenuType.js

```diff
@@ -1,19 +1,40 @@
 import get from "lodash/get";
 
-export const SHORT_MENU = "short";
-export const FULL_MENU = "full";
+import { themeShortMenu, themeFullMenu } from "../../../version/config/theme.config";
 
-/**
- * @author Bogdan Shcherban <bsc@piogroup.net>
- * @param {string} currentPathname
- * @return {string}
- */
-export function getMenuType(currentPathname) {
+const defaultShortMenu = [
+    { url: '/', label: 'Charts' },
+    { url: '/patients', label: 'Patients' }
+];
+
+const defaultFullMenu = [
+    { url: '/summary', label: 'Patient Summary' },
+    { url: '/problems', label: 'Problems / Issues' },
+    { url: '/medications', label: 'Medications' },
+    { url: '/vaccinations', label: 'Vaccinations' },
+    { url: '/allergies', label: 'Allergies' },
+    { url: '/contacts', label: 'Contacts' },
+    { url: '/top3Things', label: 'TopThreeThings' },
+];
+
+function isUrlPresented(currentResource, menuItemsArray) {
```

Comment:
what does that name mean?

Reply:
I check, that current resource is presented in menu items array.
I renamed this function to "isResourcePresentedInMenu"